### ISSUE TITLE
Research solutions for CSP preventing ANDI from working

### ISSUE THREAD

**Issue by vickyszuchin** [cisagov/manage.get.gov#2341]
### Issue description

We can't use ANDI right now without using the chrome extension and there is a desire to find a better work around, so real users would be able to use ANDI without downloading the extension as well.  The chrome extension currently disables the content-security policy but we don't really want to make that a practice. We should investigate ways to fix this. 

[ANDI FAQ section](https://www.ssa.gov/accessibility/andi/help/faq.html)) mentions the following could be a fix:

- Ask the page owner to add the ANDI scripts to the “whitelist” of allowed scripts.

### Acceptance criteria

- [ ] Review ANDI faq linked above
- [ ] investigate adding ANDI scripts to our whitelist. (Try things out in code)
- [ ] Time box max of 3 days, and if not able to add ANDI to our whitelist, provide the team with your findings
- [ ] suggest alternative solutions

### Additional context
If cloud.gov hosting our system becomes a blocker, be sure to email them or go to their office hours (Alysia can share the link). 
[Slack discussion](https://cisa-corp.slack.com/archives/C05BGB4L5NF/p1718737644934719) from dev channel
[Slack discussion](https://cisa-corp.slack.com/archives/C06UJSC6G91/p1718654447888429) from design channel
